### DISCUSSION TITLE
No configuration received within 10 seconds, communication failed

### DISCUSSION THREAD

**Discussion by 16953115** [iNavFlight/inav#8296] [Q&A]
Hello all。I try to flash Inav to my board with F722 ( it works well in betaflight) , but after flash successful, the configurator can not connect it. I have tried to  reinstall STM VCP drivers，still can not connect . But my board with F405 can work well. I dont know why my F722 can not work with Inav.Hope your help! Thanks
![5_0](https://user-images.githubusercontent.com/60212562/183255313-0417d041-b8a9-4b37-8bf4-5ea9d9aa7cf9.jpg)
![failed](https://user-images.githubusercontent.com/60212562/183255330-33c8f09f-7320-418c-b222-a5f0bd6833e3.jpg)
![20220806232521](https://user-images.githubusercontent.com/60212562/183255420-13a0b0bd-f4b0-4b76-955d-9472b0446937.png)

**Comment by erstec** [comment]
@16953115 Probably you flashed wrong target

**Reply by 16953115** [reply]
the board is my own made.I use my target can not connect it,but the led_0 and led_1 on the board works,seems the problem is just the Configurator can not communicate .I also tried to flash the targets of matekf722 manbaf722,the same result.But My board F405 can connect the Configurator.I cant find the reason

**Reply by erstec** [reply]
How it works for you? Just trying various targets?

If you created your own board and target and willing to have help from community here, probably, you should share more information, like schematics and/or betaflight unified target file (which works well for you) and target you created in INAV.

Be sure, there is not Configurator issue if it can't connect to your board, but 100% FC.

**Reply by 16953115** [reply]
Thanks very much for helps . The board that i test now is a minimum single mcu system(sorry,I don’t know how it called in ENGLISH, it has only mcu ,crystal ,bottom of boot0 ,usb on the board,and all pins all connect to pin headers)
![sch_F405](https://user-images.githubusercontent.com/60212562/183591914-dbb78b4b-29fc-4bd1-9486-517694d2c663.jpg)
![sch_f722](https://user-images.githubusercontent.com/60212562/183591996-a6fa2a9b-a768-438d-8e8d-a65867057dff.jpg)
![board](https://user-images.githubusercontent.com/60212562/183592034-941796a7-0248-4f1b-b33d-8b2081895a8a.jpg)

___________________________________________________________________________________________________________________________
___________________________________________________________________________________________________________________________
___________________________________________________________________________________________________________________________

My workflow on the F405:
1.	Flash firmwave ( targets use Matek F405 or others for test the board )
 
![FLOW_f405_1](https://user-images.githubusercontent.com/60212562/183592159-3cefeb68-96c6-4c87-b198-6abce41f011d.jpg)

2.	Connect the mpu , osd , ect step by step .To verify they can work.

![FLOW_f405_2](https://user-images.githubusercontent.com/60212562/183592209-e56ba54c-24ad-4c79-a975-f1dc68491c02.jpg)

___________________________________________________________________________________________________________________________
___________________________________________________________________________________________________________________________
___________________________________________________________________________________________________________________________

But when i try to flash a firmware to F722 ,my work stoped in step1 : after flash successful , the configurator can not communicate to the board of F722 ( without mpu osd connected) . I also tried connect a mpu with the board , but the configurator still can not communicate with the board

![FLOW_f722_1](https://user-images.githubusercontent.com/60212562/183592348-4d41fdc7-3bee-49ef-a821-9df31f6930d1.jpg)

![FLOW_f722_2](https://user-images.githubusercontent.com/60212562/183592383-a64b9e5c-483d-42eb-9df5-f488358dc312.jpg)


___________________________________________________________________________________________________________________________
___________________________________________________________________________________________________________________________
___________________________________________________________________________________________________________________________

But this board of F722 works fine with Betaflight . all peripherals (mpu ,baro , osd ,ect) work well .

![f722_workOnBF_EN](https://user-images.githubusercontent.com/60212562/183592487-ea2cb8d7-468e-4a87-83b2-90af5fefed5b.jpg)


My board of F405 works well with Inav and Betaflight . My board of F722 can only work with Betaflight , but it did not work with Inav.( The configurator can not conmunicate with it . times out )
I have tried days by days . I can not find out the reason . It makes me feel so sad.

**Reply by erstec** [reply]
Ok, one more time, let's clarify that: **there is nothing to do from Configurator side**, it works perfect. **Problems are on you builded FW**, resources and etc. FW just can't start properly because of misconfiguration. Probably I see issue, but we need to have live communication to test.

Please contact me on Discord directly and we will try to fix it.
